semver.md

```diff
@@ -286,6 +286,19 @@ modification depends on whether you updated your dependencies in order to fix
 a bug or introduce new functionality. I would usually expect additional code
 for the latter instance, in which case it's obviously a minor level increment.
 
+#### Special note for JVM or similar environments
+
+In JVM or other environments where only one version of a library could 
```

Comment:
Just remove "JVM or other", and I think it works.. The SemVer spec implicitly covers what this FAQ change makes explicit.